### DISCUSSION TITLE
what about making it on micropython on ESP32???

### DISCUSSION THREAD

**Discussion by Manudulux** [alexanderlavrushko/BLE-HUD-navigation-ESP32#4] [Ideas]
This project is so amazing. Thanks for sharing.
I thought I will try to make it in pure Python. I am a really newbie in ESP32 and in mircopython and ubluetooth is still in development stage. Anyway I gave it a try and wrote some codes to scan devices, search for their characteristics.... 
Here it is : https://github.com/Manudulux/bluetooth    
This is still work in progress. Not too sure if I am close or not to make it work fully (at least link to the iphone and get the feed on the ESP). 
Anyway I would welcome any help.

**Comment by alexanderlavrushko** [comment]
At the moment I have zero knowledge of Python and micropython for ESP32. I'm sorry I cannot help.

**Reply by Manudulux** [reply]
Hi Alexander, no problem and thanks for your contribution. Maybe someone else will join me on this project.
Take care.
